ElastosBountyProgram/back-end/src/service/CommentService.ts

```diff
@@ -150,6 +150,11 @@ export default class extends Base {
         } = param
 
         let ownerSubject = `Someone has commented on your ${type}`
+
+        if (name) {
+            ownerSubject = `${name} ${type}`
```

Comment:
I dont like that this removed the 'Someone has commented on' bit, which is what the email is about essentially.
I would make the ```name``` part of body, not subject.